.github/workflows/deploy.yml

```diff
@@ -0,0 +1,54 @@
+name: Appium Deploy
+on:
+  push:
+    tags:
+      - appium@2*
+  workflow_dispatch:
+
+
+
+permissions:
+  contents: read
+
+jobs:
+  check:
+    name: Check
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install Appium
+        run: npm i -g appium@next
```

Comment:
what if this script runs on a git tag `appium@2.3.1`? the npm `next` tag might still point to something else? do we not want to run `npm i -g appium@${{ tag }}` assuming we can get the tag as a variable?

Reply:
`tag` is only defined when the pipeline is triggered by a new tag. It is going to be unset if the pipeline is triggered manually though

Reply:
changed to use the current tag